src/evaluation_system/tests/mocks/dummy.py

```diff
@@ -40,9 +40,9 @@ class DummyPlugin(PluginAbstract):
         InputDirectory(name="input", help="An input file"),
         String(name="variable", default="tas", help="An input variable"),
     )
-    _runs = []
+    _runs = []  # type: ignore
```

Comment:
If mypy complains about it, we need to fix it. This can be done by defining a type.

Reply:
yes, let me do it in next PR